packages/shared/src/lib/reminderFields.ts

```diff
@@ -5,6 +5,16 @@ export interface ReminderFields {
     reminderOption?: string;
 }
 
+const thanksgivingUsa = (year: number = new Date().getFullYear()): Date => {
+    const novemberFirst = new Date(year, 10, 1).getDay();
+
+    return new Date(year, 10, 22 + ((11 - novemberFirst) % 7));
+};
+
+const givingTuesday = (year: number = new Date().getFullYear()): Date => {
```

Comment:
Given that we're now calculating giving Tuesday dynamically, would it be worth also building the reminder fields which we pass to the API dynamically (for example, perhaps we can assume the reminder option should always be `giving-tuesday-<year>`?)

Reply:
We should definitely do this! I'll make the change!